lib/project_types/node/commands/create.rb

```diff
@@ -65,17 +65,17 @@ def self.help
       private
 
       def check_node
-        version, stat = @ctx.capture2e(%w(node -v))
+        version, stat = @ctx.capture2e('node -v')
```

Comment:
It needs to be an array though, either `*%w(node -v)` or `['node', '-v']` whichever you prefer.

Reply:
ah crud why does it work then that's the real question but yes okay!